src/QMCWaveFunctions/lcao/RadialOrbitalSetBuilder.h

```diff
@@ -400,11 +397,11 @@ class RadialOrbitalSetBuilder: public MPIObjectBase
   void RadialOrbitalSetBuilder<COT>::addGaussian(xmlNodePtr cur)
   {
     int L= m_nlms[1];
-    using gto_type=GaussianCombo<double>;
+    using gto_type=GaussianCombo<ValueType>;
```

Comment:
Using double is on purpose. When constructing splines in the mixed precision build, Gaussian are always initialized in double. After the spline parameters are constructed, the evaluation goes to lower precision.

Reply:
Probably it should be replace with OHMMS_PRECISION_FULL

Reply:
Special handling for extra precision in mixed precision builds should be commented. A perfect example of something subtle but important ("Always use full precision here since the fudge function is numerically sensitive in large supercells") that will be not obvious to anyone but the original author.